vendor/github.com/aws/aws-sdk-go/aws/client/client.go

```diff
@@ -15,6 +15,12 @@ type Config struct {
 	Endpoint      string
```

Comment:
can't we remove those deps from the repo? I know we don't have them in df-runner. Imo they just pollute the repo.

Reply:
they're used for template functions, but if they're not needed I'll remove them